features/car/conditional_restrictions.feature

```diff
@@ -353,6 +353,18 @@ Feature: Car - Turn restrictions
             | b    | a  | bj,jc,jc,aj,aj |
             | b    | d  | bj,jd,jd       |
 
+        # here we check that conditional restrictions can be updated via re-run of `contract`/`customize` with updated `--parse-conditionals-from-now`
+        # 10am utc, sat 
+        When I run "osrm-contract  {osm_file} --time-zone-file=test/data/tz/{timezone_names}/guinea.geojson --parse-conditionals-from-now=1494064800"
+        When I run "osrm-customize {osm_file} --time-zone-file=test/data/tz/{timezone_names}/guinea.geojson --parse-conditionals-from-now=1494064800"
```

Comment:
Also probably we should vendor geojson with time zones to the repo, not sure what can be problem here, but it would ease a lot usage of this feature for beginners.